packages/base/card-api.gts

```diff
@@ -81,6 +81,7 @@ type Setter = { setters: { [fieldName: string]: Setter } } & ((
 
 interface Options {
   computeVia?: string | (() => unknown);
+  isUsed?: true; // TODO this is a workaround for a bug that needs researching
```

Comment:
filed in linear as https://linear.app/cardstack/issue/CS-5692/async-computed-sometimes-not-recomputing